res/Sectors/M1105/Gzirr!k'l.txt

```diff
@@ -0,0 +1,560 @@
+1825 Bweweprockia         D866300-9 Ga Lo                                 - - - 700   Na K1 IV M4 V      
+1921 K'greellu            C99A1PK-5 Lo Wa                                 - - - 900   Kk K6 V            
+1922 Gnakgrai             D4252PK-7 Lo                                    - K - 900   Kk F9 V M8 V       
+1924 Starev               D8C3799-A Fl                                    - - - 514   Na M1 V            
+1927 Bjejox               A260675-B De Ni Ri                              - - - 904   Kr F0 V M5 V       
+1929 Kragrenk             E225000-0 Ba                                    - - R 010   Kr M0 V M8 V M5 V  
+2025 Thoswed              E9B5000-0 Ba Fl                                 - - R 022   Kr G1 V            
+2026 Ybrofos              A254740-D Ag                                    - N - 105   Kr M9 II K4 V      
+2029 Cruhados             C758545-8 Ag Ni                                 - - - 814   Kr M6 V M3 V       
+2121 Xkarrghatii          C898324-5 Lo                                    - - - 602   Kk G4 V            
+2222 Athabbus             E533000-0 Ba Po                                 - - - 004   Na K2 V M3 V       
+2223 Snognex              C78A677-D Ni Ri Wa                              - - - 804   Na M1 V M9 V M1 V  
+2224 Slovlagral           D646677-8 Ag Ni                                 - - - 702   Na F3 V            
+2225 Uzepwot              A788753-D Ag Ri                                 - N - 603   Kr F3 III          
+2226 Gicung               X9B9332-4 FL Lo                                 - - R 812   Kr K1 V            
+2229 Tegangus             A6458A9-C Pi Ph Pa                              - - - 314   Kr M1 V            
+2230 Bj'jiked             E334A98-C Hi                                    - - - 803   Kr G3 V            
+2321 Gnanettos            C7559A9-6 Ga Hi                                 - - - 123   Na K4 V M2 V       
+2323 Ha Alhu Taey         C3207C9-9 He De Pi Na Po                        - - - 300   Na M5 V M2 V       
+2325 Greraneck            C44285A-A He Pi Po Ph                           - - - 513   Na M2 III          
+2329 Deton Diadem         E000000-0 As Ba Va                              - - - 014   Na M2 V M2 V       
+2330 Ukallia              C884376-B Lo                                    - - - 924   Na M2 V M7 V M2 V  
+2424 Kolev                E324755-9 Pi                                    - - - 813   Na G6 V M5 V       
+2425 Qyrebia              C556A88-8 Hi                                    - - - 800   Na A4 III G1 V     
+2428 Prozlang             B756489-B Ni Ga Pa                              - - - 122   Na M2 II M5 V      
+2429 Blablandee           B653874-9 Po Ph                                 - - - 814   Na M4 V            
+2522 Bjorefell            D210444-9 Ni                                    - - - 512   Na G8 V            
+2525 Harugruck            A878999-D Hi In                                 - N - 813   Nv K5 V            
```

Comment:
Is this Allegiance code supposed to be "Hv"? If not, what is the "Nv" Allegiance?

Reply:
Not got to round to the sanity checking of Gzirr!k'l yet, but I'll check this as well.

Reply:
Fixed